CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This project adheres to [Semantic Versioning](http://semver.org)
 
+## Unreleased
+Fix:
+ - Add `--version` to the CLI
```

Comment:
could you add the `improvement` `reduce RSpec runner verbosity` that was merged on the 19th of February ?